symtabAPI/src/Object-elf.h

```diff
@@ -91,6 +93,50 @@ class Object;
 class ObjectELF;
 class InlinedFunction;
 
+#ifndef PT_RISCV_ATTRIBUTES
```

Comment:
Include these in the note in DyninstElfUtils.cmake what needs to be modified when upgrading to a newer version of elfutils.

Reply:
Done